python/understack-workflows/tests/test_topology.py

```diff
@@ -1,29 +1,58 @@
-import json
-import pathlib
-
+from understack_workflows.nautobot_device import NautobotDevice
+from understack_workflows.nautobot_device import NautobotInterface
 from understack_workflows.topology import pxe_interface_name
 from understack_workflows.topology import switch_connections
 
-
-def read_json_samples(file_path):
-    here = pathlib.Path(__file__).parent
-    ref = here.joinpath(file_path)
-    with ref.open("r") as f:
-        return json.loads(f.read())
-
-
-NAUTOBOT_DATA = read_json_samples("json_samples/nautobot_device_data.json")
+NAUTOBOT_DATA = NautobotDevice(
```

Comment:
I see the same device above. Can we not share this and put it in a fixture so that it's not open coded?

Reply:
Done in 1f3cc8b